### DISCUSSION TITLE
Best slinger settings for SB500 when accessing remotely

### DISCUSSION THREAD

**Discussion by tryin100** [GerryDazoo/Slinger#643] [Q&A]
I am uncertain where to set these parameters for the best video quality when accessing a remote slinger server.
I understand there are limits to what a SB500 can do so some settings may just be overkill and others not enough.
If the recommendations are different depending on whether I am using slinger player or VLC please let me know.

Any help would be appreciated.  Thanks

Resolution=?
FrameRate=?
VideoBandwidth=?
VideoSmoothness=?

**Comment by GerryDazoo** [comment]
The short answer is it depends.
Primarily it depends on the available bandwidth for your connection to/from the Server.
So if you have a good Internet connection you might be able to use the same settings as on your LAN. But if you're zipping down I95 with a cellular data connection you might want to use a lower bandwidth configuration. This is what I use for a cellular config
Resolution=5
FrameRate=60
VideoBandwidth=1000


The setting should be the same for SlingerPlayer or VLC as they both use the same FFMPEG libraries.

**Reply by tryin100** [reply]
Excellent, thanks.   
Can you tell me what the settings would be with a good home internet connection (not fiber, but solid upload/download speeds) on the server side and on the remote player side a strong home wifi signal?

**Reply by garabik** [reply]
In my experience (mostly from the good ol' days before the Sling servers shutdown), with the resolution 640×480 (i.e. 5 in Slinger), the video stream will only rarely drop below 1Mbps, even if you specify lower bandwidth. You have to cut down the resolution if your network speed is lower.

OTOH, at least when using h264 encoding, 640×480 and 1Mbps bandwidth, rapidly changing picture (think "action movie") will saturate the bandwidth and the slingbox will compensate by dropping framerate, which looks rather bad. To be on the safe side, use 1.5Mbps (or automatic resolution, if you are using the original 1.5 player - but I think it uses way too bigger bitrate than necessary, it will happily drop to 320×240 while keeping 1Mbps bitrate).

On the third hand, if the speed is OK but the network drops packets frequently (not much the case nowadays, but can happen with e.g. poor wifi signal - cellular in my experience does not drop packets, but buffer the connection), UDP streaming with slingbox player 2.0 and a two second buffer worked really well (if left to automatic stream, it dropped the resolution instantly on a lost packet, and slowly recovered). However, with TCP, such a situation is... not good. And Slinger does not support UDP streaming.

All this is relevant if you worry about your connection not being fast/reliable enough. Seeing you have SB500, you likely want better resolution (640×480 looked perfect at the time of 17" CRT monitors, not so good on wall mounted 42" TV :-)), and networks are much better and faster. Scale your numbers accordingly...

**Comment by tryin100** [comment]
For all - here was my experience with config settings for a Sling500.  I have several Sling500s.  
If anyone has comments or suggestions I'd be happy to hear them.  Here is what worked for me:

SETUP:
Slinger server and SB500s were on the same LAN with very good speeds, on an internet connection that was very high speed and solid.  There were two remote video locations in completely different geographic locations streaming from two SB500s on my LAN.
Slinger Player was used at all viewing locations. 

CONFIGS:
(A) On both the LAN with local viewing and at remote locations these settings worked well streaming for several hours BEFORE the super bowl.

Resolution=12
FrameRate=60
VideoBandwidth=4096
VideoSmoothness=63
IframeRate=5
AudioBitRate=32

(B) When the superbowl started the remote viewing locations  immediately started having very bad choppy video and audio.  Unwatchable.
I then changed the config to the following and the remote location video and audio was very good and never stuttered or dropped. 

Resolution=5
FrameRate=60
VideoBandwidth=2048
VideoSmoothness=63
IframeRate=5
AudioBitRate=16

**Comment by Cibinium** [comment]
Hi All,

I have Slingbox 500 and I can't add any remote (any bin file), the SlingPlayer Desktop can't do to finish the setup because for 500 it's need the original Sling servers, I think. 
For Slingbox M1 I was able to do with SlingPlayer Desktop.

Any tips or tricks ?

Thank you.

**Reply by GerryDazoo** [reply]
As far as I know the procedure should be the same for the 500 and the M1. When your using SlingPlayer it will sometime look like it's just spinning it's wheels. Give it some time for the code to timeout trying to access the servers and move on to the next step.
